cli/status/lua/instance_state.lua

```diff
@@ -0,0 +1,57 @@
+local ok, config = pcall(require, 'config')
+if ok and config ~= nil and type(config) == "table" and type(config.info) == "function" then
+    config = config:info()
+else
+    config = {
+        status = "uninitialized",
+        alerts = {},
+    }
+end
+
+local config_info = {
+    status = config.status,
+    alerts = {}
+}
+for _, alert in pairs(config.alerts) do
+    table.insert(config_info.alerts, {
+        type = alert.type,
+        message = alert.message
+    })
+end
```

Comment:
AFAIS, this code just cuts off extra fields. It would be nice to a have some remark, why we need to do it.

Reply:
Added.